pubspec.yaml

```diff
@@ -0,0 +1,93 @@
+name: flutter_minimal_integration
+description: Flutter minimal intergration app
+
+# The following line prevents the package from being accidentally published to
+# pub.dev using `flutter pub publish`. This is preferred for private packages.
+publish_to: 'none' # Remove this line if you wish to publish to pub.dev
+
+# The following defines the version and build number for your application.
+# A version number is three numbers separated by dots, like 1.2.43
+# followed by an optional build number separated by a +.
+# Both the version and the builder number may be overridden in flutter
+# build by specifying --build-name and --build-number, respectively.
+# In Android, build-name is used as versionName while build-number used as versionCode.
+# Read more about Android versioning at https://developer.android.com/studio/publish/versioning
+# In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
+# Read more about iOS versioning at
+# https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
+version: 1.0.0+1
+
+environment:
+  sdk: ">=2.17.6 <3.0.0"
+
+# Dependencies specify other packages that your package needs in order to work.
+# To automatically upgrade your package dependencies to the latest versions
+# consider running `flutter pub upgrade --major-versions`. Alternatively,
+# dependencies can be manually updated by changing the version numbers below to
+# the latest version available on pub.dev. To see which dependencies have newer
+# versions available, run `flutter pub outdated`.
+dependencies:
+  flutter:
+    sdk: flutter
+  permission_handler: ^10.0.0
+  bluedot_point_sdk:
+    git:
+      url: https://github.com/Bluedot-Innovation/PointSDK-Flutter-Plugin.git
+      ref: ln/new-package-name
```

Comment:
For CI, you will need to find a way to switch between the url/tag/ref and a version (e.g. ^1.0.0) depending on if it is a dowstream or the Min App build itself. 
Probably having it by default as `bluedot_point_sdk:^1.0.0` might be easier, and then if need to, replace b git, url, ref.

Reply:
As discussed yesterday, I will keep this for now and add the default `bluedot_point_sdk:^1.0.0` once we have published the plugin to pub.dev.

Reply:
since plugin PR is merged to dev/1.0.0, this ref should be pointing to that branch or a `release/1.0.0` branch instead of a personal branch.